_pages/help.md

```diff
@@ -2,3 +2,82 @@
 layout: help_landing
 permalink: /help/
 ---
+<article class="grid-container-tablet-lg tablet-lg:padding-x-0 margin-top-9 padding-bottom-1">
+  <ul class="usa-card-group grid-row tablet:flex-align-center usa-list usa-list--unstyled">
+    <li class="card">
+      <div class="grid-row flex-row tablet:flex-align-center">
+        <div class="grid-col-2">
+          <div class="usa-card__img">
+            <img alt=" " src="{{ site.baseurl }}/assets/img/help/get-started.svg">
+          </div>
+        </div>
+        <div class="grid-col-10 padding-left-1 tablet:padding-left-3">
+          <h2 class="margin-bottom-05">
+            <a href="#">Get started with login.gov</a>
```

Comment:
Am I correct in assuming this is placeholder content and not needing to be localized quite yet?

Reply:
You are correct!